src/get.js

```diff
@@ -64,12 +95,25 @@ function loadScript(url, callback, errorCallback) {
  * @param {*} opts
  * @returns {Promise}
  */
-function get(src, opts) {
+function get(src, type, opts) {
   if (isString(src)) {
+    //if single src
+
+    //check if type is given by user otherwise self set based on extension
+    if (!type) {
```

Comment:
You can extract this logic to a separate function `determineFileType()` so that this function just does one thing - single responsibility principle. 

And this current function could be simple as this:
```js
type = type || determineFileType(src);
```

And return-early to have cleaner if / else ladder. 
```js
function determineFileType(src) {
  if (extension === '.js') {
    return 'text/javascript';
  }
  
  if (extension === '.css') {
    return 'text/css';
  }

  return null;
}
```